src/widget/flex.rs

```diff
@@ -200,9 +211,17 @@ impl<T: Data> Widget<T> for Flex<T> {
             child.widget.set_layout_rect(rect.with_origin(pos));
             major += self.direction.major(rect.size());
         }
+
+        if log::log_enabled!(log::Level::Warn) {
```

Comment:
I appreciate these sorts of diagnostics.

I don't know if it's worth doing the `log_enabled!` check, here; the comparison below is very cheap, but 🤷‍♂